docs/attributes-dictionary/http.md

```diff
@@ -0,0 +1,53 @@
+<!--- Hugo front matter used to generate the website version of this page:
```

Comment:
Would be really cool if we can make a page in the website where people could search for attributes, as in https://opentelemetry.io/ecosystem/registry/. @chalin do you think we can do this in the website from these new conventions here? Would the website need a new field for it? (type, for ex?)

Reply:
I guess that what you want is a list of attributes that link back to the page that they are defined on?

Maybe the tags feature of Hugo & Docsy might be useful in this case: https://www.docsy.dev/docs/adding-content/taxonomy/

If you'd like to explore that, please create an issue in the OTel website repo so that the request gets broader visibility, and we can assess relative to other requests. Thanks!

Reply:
@chalin What we want here is actually something very similar to this: https://uptrace.dev/opentelemetry/attributes.html

Is there a way to do something similar?